web/src/lottie-player.model.ts

```diff
@@ -0,0 +1,98 @@
+import { LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+
+// Define library version
+interface LibraryVersion {
+  THORVG_VERSION: string
+}
+
+// Define file type which can be exported
+enum ExportableType {
+  GIF = 'gif',
+}
+
+export abstract class LottiePlayerModel extends LitElement {
+  @property({ type: Boolean })
+  public direction: number = 1;
+
+  @property({ type: Number })
+  public speed: number = 1.0;
+
+  @property({ type: Number })
+  public totalFrame?: number;
+
+  @property({ type: Number })
+  public currentFrame?: number;
+
+  @property({ type: Boolean })
+  public loop: boolean = false;
+
+  /**
+   * Start playing animation.
+   */
+  public abstract play (): void
+
+  /**
+   * Pause animation.
+   */
+  public abstract pause (): void
+
+  /**
+   * Stop animation.
+   */
+  public abstract stop (): void
+
+  /**
+   * Start playing animation.
+   */
```

Comment:
missing "param" info

Reply:
The method `play()` doesn't have any params. should we consider `@param` for void parameter?

```ts
public play(): void
```

Reply:
Please check load()